deploy-board/deploy_board/templates/hosts/hosts.tmpl

```diff
@@ -254,11 +254,15 @@
             <form id="terminateSelectedHostForm" class="form-horizontal"  method="post" role="form"  action="/env/{{ env.envName }}/{{ env.stageName }}/terminate_hosts/">
                 <div class="modal-header">
                     <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
-                    <h4 class="modal-title">Host(s) Termination Confirmation</h4>
+                    <h4 class="modal-title">Host(s) Termination Confirm</h4>
                 </div>
                 <div class="modal-body">
                     <p> Are you sure you would like to terminate the following hosts?</p>
                     <input id="hostsList" class="form-control" name="hostIds" type="text" value=""/>
+                    <br>
+                    <input type="checkbox" id="replace_host" name="replaceHost" checked> Check to replace host (Uncheck to to decrement desired capacity)
+                    <br><em>Note replacement will only work if the host belongs to an auto scaling group.</em>
+                    <br><em>If you choose to not replace host, it can't be guarenteed that the auto scaling group's min and max sizes will keep identical.</em>
```

Comment:
```suggestion
                    <br><em>If you choose to not replace host, it can't be guaranteed that the auto scaling group's min and max sizes will keep identical.</em>
```